src/UserManager.ts

```diff
@@ -45,6 +46,15 @@ export type SigninPopupArgs = PopupWindowParams & ExtraSigninRequestArgs;
  */
 export type SigninSilentArgs = IFrameWindowParams & ExtraSigninRequestArgs;
 
+/**
+ * @public
+ */
+export type SigninCredentialsArgs = {
```

Comment:
now we can reuse the type from OidcClient.ts and do a small rename too:
```export type SigninResourceOwnerCredentialsArgs = ProcessResourceOwnerPasswordCredentialsArgs;```

Reply:
Fixed.